packages/chrome/src/elements/subnav/CollapsibleSubNavItem.tsx

```diff
@@ -18,8 +18,15 @@ import {
 
 export interface ICollapsibleSubNavItemProps
   extends Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'onChange'> {
+  /** Sets the header */
   header?: React.ReactNode;
+  /** Determines if the item is expanded */
   isExpanded?: boolean;
+  /**
+   * Handles item expansion changes
+   *
+   * @param {boolean} isExpanded Determines if the item is expanded
```

Comment:
The parameter is an indicator of expansion state:

> ~Determines~ Indicates if the item is expanded

Reply:
Fixed in latest update. Should we use Indicates (in general) for all boolean descriptions?